builder/converter/sectioned_view.py

```diff
@@ -0,0 +1,184 @@
+import os
+from column_mapping import mapping
+import xml.etree.ElementTree as ET
+
+
+def convert_to_xml(yaml_dict):
+    root = get_xml_from_template('sectioned_view_template.xml')
+    set_name(root, yaml_dict)
+    set_sections(root, yaml_dict)
+    return root
+
+
+def get_xml_from_template(template_name):
+    current_dir = os.path.dirname(os.path.realpath(__file__))
+    template_path = os.path.join(current_dir, 'templates/%s' % template_name)
```

Comment:
I think this can be:
`template_path = os.path.join(current_dir, 'templates', template_name)`

We can let os.path.join do all the work instead of having forward slashes in there. I think there are a few other places we have this debt.